python/aigverse/abc/_runner.py

```
@@ -0,0 +1,281 @@
+                use_init_file=use_init_file,
+                cwd=scratch,
+                binary=executable,
+            )
+
+    argv = [str(executable)]
+    if not use_init_file:
+        argv.append("-s")
+    argv += ["-q", command]
+
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)